Description: Patch fixing FTBFS #615780
 This patch was introduced to provide -lX11 linking which
 was causing a FTBFS
Author: Bart Matens <bartm@debian.org>
Last-Updated: 02 October 2011
--- ../orig/surf-0.4.1/./Makefile	2010-06-08 09:06:41.000000000 +0200
+++ ./Makefile	2011-10-02 20:11:01.000000000 +0200
@@ -26,7 +26,7 @@
 
 surf: ${OBJ}
 	@echo CC -o $@
-	@${CC} -o $@ surf.o ${LDFLAGS}
+	@${CC} -o $@ surf.o ${LDFLAGS} -lX11
 
 clean:
 	@echo cleaning
